cql3/expr/expression.hh

```diff
@@ -610,7 +610,6 @@ extern expression search_and_replace(const expression& e,
         const noncopyable_function<std::optional<expression> (const expression& candidate)>& replace_candidate);
 
 extern expression prepare_expression(const expression& expr, data_dictionary::database db, const sstring& keyspace, lw_shared_ptr<column_specification> receiver);
-extern expression prepare_expression_multi_column(const expression& expr, data_dictionary::database db, const sstring& keyspace, const std::vector<lw_shared_ptr<column_specification>>& receivers);
```

Comment:
The commit message is not descriptive enough. All I see here is code being deleted - but I don't know why (what it was used for before, and why it's no longer needed).